README.md

```diff
@@ -210,62 +216,148 @@ The basic properties that must be configured for the connector are:
-| client.push.full.wait             | Sets whether Ably should wait for all the effects of push REST requests before responding.                                                                                                                                      | *Boolean* ||
-| client.tls                        | Sets whether TLS is used for all connection types.                                                                                                                                                                              | *Boolean* | True    |
-| client.token.params               | Sets whether the configured token parameters are used.                                                                                                                                                                          | *Boolean* ||
-| client.token.params.capability    | Stringified JSON capability requirements for the token. When omitted, the REST API default to allow all operations is applied by Ably, with the string value `{“*”:[“*”]}`. Requires `client.token.params` to be set to `true`. | *String*  ||
-| client.token.params.client.id     | The client ID to include with the token. Requires `client.token.params` to be set to `true`.                                                                                                                                    | *String*  ||
-| client.token.params.ttl           | The requested time to live (TTL) for the token in milliseconds. When omitted, the REST API default of 60 minutes is applied by Ably. Requires `client.token.params` to be set to `true`.                                        | *Boolean* | 0       |
-| client.loglevel                   | Sets the verbosity of logging.                                                                                                                                                                                                  | *Integer* | 0       |
+| Property                          | Description                                                                                                                                            | Type      | Default |
+|-----------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|---------|
+| message.name                      | Ably message name to publish. Can be a pattern, as per `channel` above.                                                                                | *String*  |         |
+| batchExecutionThreadPoolSize      | The maximum number of parallel outgoing REST API requests to publish content to Ably                                                                   | *Integer* | 10      |
```

Comment:
For all of these limits are there any additional maximums and minimums to what these values can be?

Reply:
Done in https://github.com/ably/kafka-connect-ably/pull/142/commits/aba1da6aab4ddd28ec80d4ef51080c3195a73a26

Reply:
For batch publishing, you'd be limited by the maximum payload size limits here:

https://ably.com/docs/api/rest-api#batch-publish

So it would be a case-by-case basis

Reply:
Added a line about the limits: https://github.com/ably/kafka-connect-ably/pull/142/commits/162248ad6fedc2647f445a6e19b175b6740c69a9